pkgs/by-name/sc/scrutiny/package.nix

```diff
@@ -0,0 +1,71 @@
+{ buildNpmPackage
+, buildGoModule
+, fetchFromGitHub
+, nixosTests
+, lib
+, ...
+}:
+let
+  name = "scrutiny";
+  version = "0.7.2";
+
+  repo = fetchFromGitHub {
```

Comment:
```suggestion
  src = fetchFromGitHub {
```
same here basically

Reply:
This makes things complicated with infinite recursion because we need to state the path for the frontend so I've left it for now?